apps/settings/lib/Service/AuthorizedGroupService.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/**
+ * @copyright Copyright (c) 2021 Nextcloud GmbH
+ *
+ * @author Carl Schwan <carl@carlschwan.eu>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Settings\Service;
+
+use OCP\AppFramework\Db\DoesNotExistException;
+use OCP\AppFramework\Db\MultipleObjectsReturnedException;
+
+use OC\Settings\AuthorizedGroup;
+use OC\Settings\AuthorizedGroupMapper;
+use OCP\DB\Exception;
+use OCP\IGroup;
+
+class AuthorizedGroupService {
+
+	/** @var AuthorizedGroupMapper $mapper */
+	private $mapper;
```

Comment:
```suggestion
	private AuthorizedGroupMapper $mapper;
```

Wouldn't this also be possible? :-) (or do we support older PHP releases that don't support this?)

Reply:
We still have 7.3 support and that is 7.4+ only https://php.watch/versions/7.4/typed-properties

Reply:
TIL! Thanks Julius :)

Reply:
Yeah unfortunately we still live in php 7.3 so we only have phpdoc for now.